### DISCUSSION TITLE
Final project 是否需要合成

### DISCUSSION THREAD

**Discussion by junshiuanhsieh** [bol-edu/HLS-SOC-Discussions#190] [Q&A]
由於final project 規定要使用SDRAM，但lab-sdram是不能合成的，所以想請問final project 是只需要做 rtl simulation 嗎

**Comment by junshiuanhsieh** [comment]
另外 想請問是否能自行新增新的sram，然後delay是否會有限制

**Comment by LingChiYang** [comment]
同學你好

請同學用這個thread統一針對final project的spec討論。感謝
https://github.com/bol-edu/HLS-SOC-Discussions/discussions/189

**Comment by boledulab** [comment]
@junshiuanhsieh lab-sdram 是可以合成在 FPGA 上跑。台大 TA 可以 Comment。

**Comment by john-1109** [comment]
@junshiuanhsieh 同學你好，lab-sdram是使用FPGA的BRAM當作SDRAM的bank，所以是可以合成的
不能合成的是SDRAM的behavior model
